SQLite/Core/Blob.swift

```diff
@@ -30,16 +30,16 @@ public struct Blob {
         self.bytes = bytes
     }
 
-    public init(bytes: UnsafePointer<Void>, length: Int) {
-        self.init(bytes: [UInt8](UnsafeBufferPointer(
-            start: UnsafePointer(bytes), count: length
-        )))
+    public init(bytes: UnsafeRawPointer, length: Int) {
+        // TODO correct count
```

Comment:
It works in tests and actual app, though I'm not sure whether my changes correct, maybe we could take a look at swift 2.3 branch.